l1b_lambda/app.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+from shutil import copytree
+
+from aws_cdk import App
+
+from stacks.level1b_stack import Level1BStack
+
+app = App()
+
+copytree(
+    Path("..") / "src" / "mats_l1_processing",
+    Path(".") / "level1b" / "mats_l1_processing",
+    dirs_exist_ok=True,
+)
```

Comment:
This looks to me like it solves the problem just as well as making a layer, but I could make a proper layer if that's a better solution.

Reply:
Not sure... I'll have a look, but I doubt it, unfortunately... Another version is to use the built wheel, that we already require. It is basically a zip-file, so we could unzip it in place, maybe? Or can you configure cdk to ignore `__pycache__` when synthing?

Reply:
Turned out it was easy to ignore when copying. `cdk synth` will still copy cache-files if present though, which is a nuisance that I would have thought someone had solved already, but I can't find out how to prevent it.